instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/suites/CoreFifthPartTestSuite.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.navigation.instrumentation_tests.suites
+
+import com.mapbox.navigation.instrumentation_tests.core.RoutesPreviewTest
+import com.mapbox.navigation.instrumentation_tests.core.SanityCoreRouteTest
+import com.mapbox.navigation.instrumentation_tests.core.SetRoutesTest
+import com.mapbox.navigation.instrumentation_tests.core.TripSessionsBillingTest
+import com.mapbox.navigation.instrumentation_tests.core.UpcomingRouteObjectsTest
+import com.mapbox.navigation.instrumentation_tests.core.WaypointsTest
+import org.junit.runner.RunWith
+import org.junit.runners.Suite
+
+@RunWith(Suite::class)
+@Suite.SuiteClasses(
```

Comment:
The problem I see here is that when we add a test class, we have to add it to some list. We'll definitely forget to do it. I wonder if this division by suites can de done automatically based on all the tests we have?

Reply:
hmm good idea, We can try to create suites dynamically before assembling instrumentation tests